### DISCUSSION TITLE
Missing Quality Filter in Settings

### DISCUSSION THREAD

**Discussion by makako-dev** [photoprism/photoprism#3929] [Q&A] [answered]
I want to turn off the quality filter per [this](https://docs.photoprism.app/user-guide/organize/review) guide "(unless the quality filter was disabled in [Settings](https://docs.photoprism.app/user-guide/settings/general/))", however I do not see it on the settings page. I am using most recent photoprism version as of today. Unless there is a way to approve +50K photos in my library at once, I need to turn this filter off. Any ideas?

**Comment by lastzero** [comment]
Maybe add a screenshot? Do you see it on our public demo?

**Comment by makako-dev** [comment, marked as answer]
@lastzero Actually never mind. I was looking at the General tab but just realized it was on the Library tab. Thanks for your quick reply!
